server/migrations/20230826123551-add-description-and-banner-to-channels.js

```diff
@@ -0,0 +1,19 @@
+exports.up = async function (db) {
+  await db.runSql(`
+    ALTER TABLE channels
+    ADD COLUMN description text,
+    ADD COLUMN banner_id uuid;
```

Comment:
This shouldn't have a banner ID any more, yeah?

Reply:
I actually left it in this migration and removed it in the following migration because the order in which my migrations were executed locally was fucked. But anyway, I've managed to unfuck them so this is not needed anymore.